Dockerfile.pr

```diff
@@ -39,6 +39,9 @@ ENV TAG_VERSION=$TAG_VERSION
 ENV AWS_LWA_ENABLE_COMPRESSION=true
 ENV HOSTNAME=localhost
 ENV PORT=3000
+# Use the REVIEW_ENV environment variable to determine if the app is running in a review environment
+# Enables memory caching for the prerendering of pages
+ENV REVIEW_ENV=1
```

Comment:
so if it is `1` what does it mean exactly? if it represents a boolean truth,  later on I see the following check which is confusing: `process.env.NODE_ENV === "production" &&
    process.env.REVIEW_ENV`

Reply:
I can simplify it for now by removing the NODE_ENV check until we also include the external caching in non-PR-Review environments.
The "1" simply assigns an arbitrary value that results in a Boolean 'true' to check if the variable is defined.  I could also assign the value to 'TRUE' if it helps with readability.

Reply:
NODE_ENV = production also when you build and run the app locally.

Reply:
Will create a new PR with the changes